src/components/test/Thumbnail-test.js

```diff
@@ -28,7 +28,7 @@ describe('Thumbnail', () => {
   context('when in loading state', () => {
     it('renders a loading spinner', () => {
       const wrapper = createComponent({ isLoading: true });
-      assert.isTrue(wrapper.find('SvgIcon[name="spinner"]').exists());
+      assert.isTrue(wrapper.find('Spinner').exists());
```

Comment:
Yeah, I can't make a good excuse as to why `Thumbnail` was testing for the presence of an icon instead of the Spinner component. It may have been that I implemented `Thumbnail` before `Spinner` was a component (i.e. it existed as a pattern but hadn't been component-ized yet).